### ISSUE TITLE
Improve accessibility for screen readers

### ISSUE THREAD

**Issue by Lorik-debug** [Lorik-debug/crypto_bus_DESC#30]
Some UI elements lack proper ARIA attributes and semantic HTML. Improve accessibility to enhance the experience for visually impaired users.

**Comment by Lorik-debug** [comment]
ок
